scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -622,11 +668,12 @@ object Execution {
                   Future.successful(Map.empty)
               }
 
-            failFastZip(otherResult, localFlowDefCountersFuture).map {
-              case (lCounters, fdCounters) =>
-                val summedCounters = (fdCounters :: lCounters).reduce(_ ++ _)
-                (fn(conf, mode), summedCounters)
-            }
+            val bothFutures = failFastZip(otherResult, localFlowDefCountersFuture)
```

Comment:
So this `localFlowDefCountersFuture` is a `Future` for the stats from finishing the cascading flow?